libipl/p10/common.C

```diff
@@ -287,3 +290,46 @@ int ipl_set_sbe_state_all_sec(enum sbe_state state)
 	}
 	return ret;
 }
+
+void ipl_process_fapi_error(const fapi2::ReturnCode& fapirc,
+		struct pdbg_target *target, bool deconfig)
+{
+	if(fapirc == fapi2::FAPI2_RC_SUCCESS) {
+		ipl_error_callback(IPL_ERR_OK);
+	} 
+	else if(fapirc.getCreator() == fapi2::ReturnCode::CREATOR_PLAT) {
+		CDG_Target cdgTarget;
+		cdgTarget.callout_priority =
+		fapi2::plat_CalloutPriority_tostring(fapi2::CalloutPriorities::MEDIUM);
+
+		ATTR_PHYS_BIN_PATH_Type physBinPath;
+		uint32_t binPathElemCount = dtAttr::fapi2::ATTR_PHYS_BIN_PATH_ElementCount;
+		if (!pdbg_target_get_attribute(target, "ATTR_PHYS_BIN_PATH",
+			std::stoi(dtAttr::fapi2::ATTR_PHYS_BIN_PATH_Spec),
+				binPathElemCount, physBinPath)) {
+			ipl_log(IPL_ERROR, "Failed to read ATTR_PHYS_BIN_PATH for target %s\n",
+				pdbg_target_path(target));
+			ipl_error_callback(IPL_ERR_ATTR_READ_FAIL);
+		}
+		else {
```

Comment:
Should we add https://github.com/open-power/libekb_p10/blob/main/hwpf/fapi2/src/plat/plat_utils.C#L233 BMC0004 procedure calolut also for FSI failures?

Reply:
yes we can